apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -127,10 +126,12 @@ export default class AnimationPickerBody extends React.Component {
   };
 
   animationCategoriesRendering() {
-    return Object.keys(this.props.categories).map(category => (
+    const libraryManifest = this.props.getLibraryManifest();
+    const categories = libraryManifest.categories;
+    return Object.keys(categories).map(category => (
       <AnimationPickerListItem
         key={category}
-        label={this.props.categories[category]}
+        label={msg[`animationCategory_${category}`]()}
```

Comment:
Is it worth having a default value in case the category doesn't map to a string? Something like ```msg[`animationCategory_${category}`]() || category```?